pages/enterprise/intro.rst

```diff
@@ -4,8 +4,41 @@ Introduction
 
 .. _enterprise_features:
 
-Graylog Enterprise, built on top of the Graylog open source platform, offers
-additional features that enable users to deploy Graylog at enterprise scale
-and apply Graylog to processes and workflows across the whole organization.
+Graylog Enterprise is made of the combination of the open source core and several plugins that contribute functionality. This way someone coming from open source can add Enterprise simply by installing a new operating system package.
 
-Please see the `Graylog Enterprise Page <https://www.graylog.org/enterprise>`_ for details.
+When the Graylog enterprise plugins and Graylog enterprise integrations plugin is installed the some additional features are added to Graylog. The following list should give a brief overview what is added to Graylog. All of this will only work with a valid enterprise license. 
+
+
+
+    - :ref:`Archiving<archivetoc>`
+		* Store the data after a configurable time on disk for infinite time. This enables to fulfill most compliance rules about storing specific data for a specific amount of time.
```

Comment:
Would Say:   Archiving allows you to store the data to long term retention location, for an infinite amount of time.  This can be local or removable media.  This will allow most users to meet compliance regulation around data retention.